randovania/games/fusion/gui/dialog/game_export_dialog.py

```diff
@@ -25,19 +29,69 @@ class FusionGameExportDialog(GameExportDialog, Ui_FusionGameExportDialog):
     def game_enum(cls) -> RandovaniaGame:
         return RandovaniaGame.FUSION
 
+    def __init__(self, options: Options, patch_data: dict, word_hash: str, spoiler: bool, games: list[RandovaniaGame]):
+        super().__init__(options, patch_data, word_hash, spoiler, games)
+
+        self._base_output_name = f"MARS - {word_hash}.gba"
```

Comment:
i dont like hardcoding the extension here. could you please change:
- `valid_input_file_types` to `valid_input_file_type` and make it return a string
- `valid_output_file_types` to `valid_output_file_type` and make it return a string
- use `self.valid_input_file_type` here in this line
- change the other usages in `_on_input_file_button` and `_on_output_file_button` to have a list that wrap around `self.valid_input/output_file_type`

makes more sense IMO since we dont ever really have a need to not export a gba rom file

Reply:
if it doesn't change what's wrong with hard coding it?

Reply:
because you're now hardcoding it in two places: here and the valid input file types.

Reply:
didn't we not want to change `list[str]` to `string` because it's used by `prompt_for_input_file`

Reply:
`prompt_for_input_file` just wants to take in a list. hence
> change the other usages in `_on_input_file_button` and `_on_output_file_button` to have a list that wrap around `self.valid_input/output_file_type`

Reply:
in practice it could look like this:
`input_file = prompt_for_input_file(self, self.input_file_edit, [self.valid_input_file_type])`

Reply:
you didnt resolve this